spec/solidus_retail/order/generate_pos_order_spec.rb

```diff
@@ -33,6 +33,36 @@
       subject
       expect(last_order).to be_complete
     end
+
```

Comment:
Everything is mocked besides the data right? We care about the end result being a complete and whole Solidus Order right? Not a bunch of hypothetically mocked items in the Order correct? That seems brittle to me